docs/docs-content/vm-management/vm-migration-assistant/migrate-vms-vmo-cluster.md

```diff
@@ -0,0 +1,929 @@
+---
+sidebar_label: "Migrate VMs to a VMO cluster"
+title: "Migrate VMs to a VMO cluster"
+description: "Learn how to migrate VMs to Palette VMO using the VM Migration Assistant"
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vmo", "vm migration assistant"]
+---
+
+Follow this guide to create source providers and migrate your VMs to your VMO cluster.
+
+## Limitations
+
+- You can only migrate VMs hosted in VMware vSphere.
+- Only VMs whose operating systems are included under
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html) can be migrated.
+
+## Prerequisites
+
+- A healthy VMO cluster. Refer to the [Create a VMO Profile](../create-vmo-profile.md) for further guidance.
+
+  - The VMO cluster must have access to VMware and the VMs you want to migrate.
+
+  :::warning
+
+  If you need to provision `Block` storage volumes during the VM migration process, add the following custom
+  configuration to your VMO cluster OS pack. Applying this configuration may cause a cluster repave. For more
+  information, refer to
+  [Repave Behaviors and Configurations](../../clusters/cluster-management/node-pool.md#repave-behavior-and-configuration)
+
+  Additionally, we recommend provisioning volumes with the `ReadWriteMany` access mode to ensure that VMs can be
+  [live migrated](https://kubevirt.io/user-guide/compute/live_migration/#limitations).
+
+  ```yaml
+  kubeadmconfig:
+     preKubeadmCommands:
+        # Start containerd with new configuration
+        - systemctl daemon-reload
+        - systemctl restart containerd
+     files:
+        - targetPath: /etc/containerd/config.toml
+        targetOwner: "root:root"
+        targetPermissions: "0644"
+        content: |
+           ## template: jinja
+
+           # Use config version 2 to enable new configuration fields.
+           version = 2
+
+           imports = ["/etc/containerd/conf.d/*.toml"]
+
+           [plugins]
+              [plugins."io.containerd.grpc.v1.cri"]
+              sandbox_image = "registry.k8s.io/pause:3.9"
+              device_ownership_from_security_context = true
+              [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+              runtime_type = "io.containerd.runc.v2"
+              [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+              SystemdCgroup = true
+  ```
+
+  :::
+
+- A VMware vSphere user account with the necessary permissions to manage the VMs you want to migrate.
```

Comment:
```suggestion
- A vCenter user account with the necessary permissions to manage the VMs you want to migrate.
```

The first question our customer will ask is what are the necessary VMware permissions required. Also, we don't need to "manage" the source VMs.